client/src/main/java/com/paypal/selion/reports/runtime/SaverFileSystem.java

```diff
@@ -131,8 +131,8 @@ public PageContents getScreenshotByName(String name) throws IOException {
     @Override
     public void init() {
         logger.entering();
-        (new File(outputFolder)).mkdirs();
```

Comment:
In case of triggering the test case execution from eclipse the output folder will be <Base directory>\test-output\

I tested the same by deleting and commenting out the code to create directory in the init() method . The folder gets created and inside it the report resources and the actual report gets populated.

In both cases i ran the suite as well as a standalone test case. The output related to reports are getting populated properly